examples/loader-rust-to-json/src/index.md

```diff
@@ -188,7 +188,7 @@ Inputs.table(hands.summary)
 
 </div>
 
-Taking advantage of Rust’s performance and simple parallelism, the data loader above was able to summarize ${hands.meta.count / 1e6} million hands in ${hands.meta.duration_s.toFixed(1)} seconds.
+Taking advantage of Rust’s performance and simple parallelism, the data loader above was able to summarize ${hands.meta.count / 1e6} million hands in ${hands.meta.duration_ms.toLocaleString()} ms.
```

Comment:
```suggestion
Taking advantage of Rust’s performance and simple parallelism, the data loader above was able to summarize ${hands.meta.count / 1e6} million hands in ${hands.meta.duration_ms.toLocaleString("en-US")} ms.
```

When interpolating localized values into English text, we should specify the locale explicitly. It’ll look weird _e.g._ to interpolate French localization into English.